addon/src/components/animated-container.hbs

```diff
@@ -1,8 +1,8 @@
 {{!--
   The @class is only there to support a deprecated usage.
 --}}
-{{#let (element this.tag) as |Tag|~}}
+{{#let (component (ensure-safe-component (-element this.tag)) tagName=this.tag) as |Tag|~}}
```

Comment:
For some reason, AST provided by `ember-element-helper` did not run.

Just a wild guess, maybe it's because this code is imported via `ember-auto-import` whereas AST is setup in Ember CLI pipeline (I'm probably wrong).

Anyways it does not matter as we agreed to get rid of the AST, there is PR for that https://github.com/tildeio/ember-element-helper/pull/82